napari/components/dims.py

```diff
@@ -207,11 +184,7 @@ def ndim(self, ndim):
             # Range value is (min, max, step) for the entire slider
             self._range = [(0, 2, 1)] * (ndim - cur_ndim) + self._range
             # Point is the slider value if in point mode
-            self._point = [0] * (ndim - cur_ndim) + self._point
-            # Interval value is the (min, max) of the slider selection
-            # if in interval mode
-            self._interval = [(0, 1)] * (ndim - cur_ndim) + self._interval
-            self._mode = [DimsMode.POINT] * (ndim - cur_ndim) + self._mode
```

Comment:
I know we don't have other modes right now, but removing these lines seems a bit dangerous? Shouldn't the point be set as well?

Reply:
If I drop the concept of modes and interval (which isn't implemented anywhere) then those two lines do nothing and aren't needed. Point doesn't need to be set, as point is never stored, it is calculated from step